### ISSUE TITLE
ledger nano s prompt to sign transaction not appearing 

### ISSUE THREAD

**Issue by cryptickt** [CityOfZion/neon-wallet#393]
Hi there, can someone please help? Im using the Ledger Nano S to connect to my Neon wallet, just yesterday I was able to send Neo to an exchange with success, and today I am trying to send Neo again to an exchange, however it keeps giving me a "transaction failed" message, wont prompt the Nano S  so that I can sign the transaction. And I do have with latest updated versions..What can I do to fix this guys?? Thanks for your assistance in advance

**Comment by stevesbrain** [comment]
@cryptickt Have you uninstalled and reinstalled the NEO app on the Ledgder? Do you have NEO app open on the Ledger before opening Neon-wallet?

**Comment by cryptickt** [comment]
Hi Steve,

I uninstalled and reinstalled neo app on Ledger, and Neon wallet on mac laptop. I was able to successfully connect the Ledger Nano S with the Neon wallet, however when confirming the transaction, typically the prompt shows up but tried this at least 10xs with no success of prompt appearing.  I have completed several transactions through the neon wallet with the Ledger and this is only the first occassion that this has occurred.  Having said all this I restarted my laptop and was able to succesfully complete my transaction, thank you for checking in, its much appreciated.



Best Regards,
Kim To

-------- Original message --------
